backend/src/routes/sessions.ts

```diff
@@ -295,6 +325,20 @@ router.patch(
         calendlyUri: newCalendlyURI,
       };
       await Session.findByIdAndUpdate(sessionId, { $set: updates }, { new: true });
+      await sendNotification(
+        "A session has been rescheduled",
+        "Your upcoming session with " +
+          mentor.name +
+          " has been rescheduled! Check out your new session details.",
+        mentee.fcmToken
+      );
```

Comment:
Same thing here. Not sure if it makes sense to send a notification to the person thats doing the rescheduling.

Reply:
Also included in the Figma